### PR TITLE
Share speech URLs on File and Channel pages

### PR DESCRIPTION
**Pull request by daovist**
New PR for #1028 @tzarebczan @liamcardenas 

This creates `<ViewOnWebButton>` with a `GLOBE` icon and uses it on `<FilePage>` and `<ChannelPage>`

Also fixes ~10 eslint errors

### CODE REVIEW

File: src/renderer/page/file/view.jsx [hunk 1]
```diff
@@ -180,6 +183,14 @@ class FilePage extends React.Component<Props> {
                       <SubscribeButton uri={subscriptionUri} channelName={channelName} />
                     </React.Fragment>
                   )}
+                  {speechSharable && (
```
[the remark this answers is not in the document]
Comment (daovist) [reply src/renderer/page/file/view.jsx]: I did, and it did not change anything

**Comment by daovist** [comment]
I am not sure what's going on with this conflict in `src/renderer/scss/component/_form-field.scss`

What Github tells me is a problem on lines 54-58 is not a problem locally

**Comment by seanyesmunt** [comment]
Can you rebase and get this up to date?

**Comment by tzarebczan** [comment]
@seanyesmunt Tired of adding buttons to the show page yet? haha Wonder how this fits in with your changes to file actions. I'll try to give it a whirl today.

**Pull request merged**